hazelcast/src/main/java/com/hazelcast/sql/impl/client/SqlClientService.java

```diff
@@ -55,7 +55,7 @@
     private static final int SERVICE_ID_MASK = 0x00FF0000;
     private static final int SERVICE_ID_SHIFT = 16;
 
-    /** ID of the SQL beta service. Should match the ID declared in Sql.yaml */
+    /** ID of the SQL service. Should match the ID declared in Sql.yaml */
```

Comment:
I see that this id is used in the `NoSuchMessageTask`. Can you check there as well? Do we still need to treat SqlMessages differently?

Reply:
TBH I'm not sure what are our guarantees... @viliam-durina could you give me a hand?

Reply:
Actually, we guarantee client compatibility in both directions since 5.0